package.json

```diff
@@ -76,6 +76,7 @@
     "rollup-plugin-license": "^2.1.0",
     "sass-loader": "^8.0.2",
     "sort-package-json": "^1.44.0",
+    "twilio-video": "^2.7.1",
```

Comment:
can we replicate fixture without this particular package?

Reply:
What do you suggest? Initially added to replicate with similar dep

Reply:
I will see if I can find another large dependency already within the Nuxt codebase as I hate the thought of adding unnecessarily to install. 🏋️‍♂️

Reply:
https://github.com/nuxt/nuxt.js/pull/7808/commits/e9f60c3b31b314245155d7b5f41fdb6aa891df74